Makefile

```diff
@@ -421,15 +429,17 @@ run-alarms-migrate: binary ##Migrate all the way up
 
 .PHONY: run-postgres
 run-postgres: ##Run O-RAN postgres
-	oc apply -k ./internal/service/postgres/k8s/overlays/dev
+	oc apply -k ./internal/service/kustomize/postgres/overlays/dev
```

Comment:
can we revert back to the original? I think the idea is that /pg is service and eventually the content under /k8s will be replaced with Go code that's called from the operator

Reply:
done